lib/hutch/worker.rb

```diff
@@ -20,6 +21,7 @@ def initialize(broker, consumers)
     # never returns.
     def run
       setup_queues
+      setup_procs.each(&:call)
```

Comment:
Why does the worker invoke those callables? Is it to make sure the queues are set up first?